pyomo/contrib/viewer/ui.py

```diff
@@ -23,46 +23,48 @@
 """
 A simple GUI viewer/editor for Pyomo models.
 """
-from __future__ import division, print_function, absolute_import
-
 __author__ = "John Eslick"
 
 import os
 import logging
+
 try:
     from IPython import get_ipython
 except ImportError:
+
     def get_ipython():
         raise AttributeError("IPython not available")
-import pyomo.contrib.viewer.report as rpt
-import pyomo.environ as pyo
 
-_log = logging.getLogger(__name__)
 
+import pyomo.contrib.viewer.report as rpt
+import pyomo.environ as pyo
 from pyomo.contrib.viewer.qt import *
```

Comment:
We've been trying to replace all uses of `import *` with explicit imports. Would you mind reworking this import statement as part of this PR?

Reply:
No problem, will do.